packages/tasit-sdk/src/Decentraland.test.js

```diff
@@ -644,14 +640,16 @@ describe("Decentraland", () => {
               argsArray
             );
 
+            // eslint-disable-next-line no-unused-vars
             const onFailed = message => {
```

Comment:
The fact that this is unused probably means we need a test that uses it? Or do we have one and this is copied and pasted in this test too?

Reply:
I'll split this test case, one using action events and other using contract events (since seems that contract events issue was solved - and that listener isn't being used because of that).

Reply:
Great, that sounds perfect

Reply:
Done on PR https://github.com/tasitlabs/TasitSDK/pull/386

Reply:
Great, thanks! Resolving this